dvc/commands/plots.py

```diff
@@ -270,8 +270,8 @@ def add_parser(subparsers, parent_parser):
     fix_subparsers(plots_subparsers)
 
     SHOW_HELP = (
-        "Generate plots from target files or plots definitions from "
-        "`dvc.yaml` file."
+        "Generate plots from target files or standalone plots configurations "
+        "from `dvc.yaml` file."
```

Comment:
Not sure I understand this change. There's nothing about `dvc plots show` that is specific to standalone plots, right? I think we could leave this as is.